compiler/include/FnSymbol.h

```diff
@@ -47,8 +49,9 @@ class FnSymbol : public Symbol {
   IntentTag                  thisTag;
   RetTag                     retTag;
 
-  // Attached original (user) iterators before lowering.
+  // Support for iterator lowering and parallel loops.
   IteratorInfo*              iteratorInfo;
+  IteratorGroup*             iteratorGroup;
```

Comment:
Let's put a comment here that says "iteratorGroup stores pointers to other iterator variants (serial, standalone, etc)"